rllib/algorithms/ddppo/ddppo.py

```diff
@@ -24,9 +24,6 @@
 from ray.rllib.algorithms.ppo import PPOConfig, PPO
 from ray.rllib.evaluation.postprocessing import Postprocessing
 from ray.rllib.evaluation.rollout_worker import RolloutWorker
-from ray.rllib.execution.common import (
```

Comment:
automated:
deprecated metrics keys should no longer be handled by users in `training_step()` methods